pkg/api-server/testdata/resources/inspect/meshgateways/_rules/meshhttproute.golden.json

```diff
@@ -220,57 +220,13 @@
    "inboundRules": [],
    "toResourceRules": [],
    "toRules": [
-    {
```

Comment:
ugh... this one is actually incorrect, with MeshHTTPRoute for MeshGateway we allow `to[].targetRef: Mesh` and combining with `__rule-matches-hash__` it behaves differently. Probably you should restore your check of number of distinct tags to be 1